src/Bicep.Core/TypeSystem/Az/OciAzTypeLoader.cs

```diff
@@ -53,7 +53,7 @@ public static OciAzTypeLoader FromTgz(string pathToGzip)
 
         protected override Stream GetContentStreamAtPath(string path)
         {
-            if (this.typesCache.TryGetValue($"./{path}", out var bytes))
+            if (this.typesCache.TryGetValue($"{path}", out var bytes))
```

Comment:
Isn't this meant to be a relative path? (The path key in https://raw.githubusercontent.com/Azure/bicep-types-az/main/generated/index.json is `"RelativePath"`.)

Reply:
It is meant to be a relative path, but as you can see `"RelativePath":"addons/microsoft.addons/2017-05-15/types.json"` the path is not including the `./` prefix in `index.json`